### ISSUE TITLE
[bug] PieChart Lables don't use the selected Theme?

### ISSUE THREAD

**Issue by hansgru** [highcharts/highcharts#2192]
It looks like the PieChart labels and lines don't use the Theme colors.
E.g. testable on your site: 
http://www.highcharts.com/demo/pie-basic/gray
looks pretty unreadable :(.

The main problem is that the "gray"  theme is used for accessibility (high contrast) setups, but it does not work correctly with all chart types :(.

**Comment by KacperMadej** [comment]
The pie demo is using the selected theme.

Please reopen in case of regression.
